packages/blade/jest.web.config.js

```diff
@@ -22,4 +22,7 @@ module.exports = {
     'jest-useragent-mock',
     './jest-setup.web.js',
   ],
+  moduleNameMapper: {
+    '\\.(css|less|sass|scss)$': '<rootDir>/src/_helpers/testing/styleMock.js',
```

Comment:
😅 Oh because for the Checkbox's keyframe animation `fadeIn` and `fadeOut` I've currently imported a plain CSS file. Could not figure out how to make it work with the styled-component's `Keyframe` 

I guess I'll have to revisit this since it could cause issues while bundling.

Reply:
Fixed! Revisted the logic and fixed it with styled component's keyframe.

Reply:
Removed.